src/Microsoft.ML.StandardLearners/Standard/Online/AveragedLinear.cs

```diff
@@ -11,7 +11,6 @@
 using Microsoft.ML.Internal.Utilities;
 using Microsoft.ML.Learners;
 using Microsoft.ML.Numeric;
-using Float = System.Single;
```

Comment:
was this change inserted to chase the review boredom :)

Reply:
:)